Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -159,6 +159,9 @@ The hosting settings allow you to host buckets in the object store on a custom D
 
 * `dnsNames`: a list of DNS names to host buckets on. These names need to valid according RFC-1123. Otherwise it will fail. Each endpoint requires wildcard support like [ingress loadbalancer](https://kubernetes.io/docs/concepts/services-networking/ingress/#hostname-wildcards). Do not include the wildcard itself in the list of hostnames (e.g., use "mystore.example.com" instead of "*.mystore.example.com"). Add all the hostnames like openshift routes otherwise access will be denied, but if the hostname does not support wild card then virtual host style won't work those hostname. By default cephobjectstore service endpoint and custom endpoints from cephobjectzone is included. The feature is supported only for Ceph v18 and later versions.
 
+!!! Note
+    The user need to confirm the port exposed by wildcard supported endpoint should be same that of rgw kubernetes service. Most provides like ingress loadbalancer or openshift routes does the same. If the SSL enabled for `CephObjectStore` then the `dnsNames` should include the endpoints which exposes the secure port and user need to update the SSL certs used by the RGW contains that domain.
```

Comment:
Could you rephrase the paragraph? I don't understand it.

Reply:
I want to cover following:

The port-defined domains in `dnsNames` accessing rgw should be same as `cephobjectstore.Spec`. By default ingress loadbalancer or openshift routes uses same port as the service which they are exposing 

If secureport is defined in `cephobjectstoreSpec`, then dnsNames contain domains which uses the secure port. And this domain need approved by the SSL certs which RGW is using

Reply:
What is a "port-defined domain"? I thought the port was independent from the dns name.
I understand better now what you are saying. Can you update the doc paragraph now instead of just the comment?